Automattic/pocket-casts-android
# Remove recent changes to display not-enough-ratings

## Description
This PR is reverting the changes from https://github.com/Automattic/pocket-casts-android/pull/1453 because there are some ongoing discussions around how to handle this, and it sounds like it will change. 

I'm removing the changes instead of adding a feature flag because it should be pretty easy to reimplement this with the proper requirements when we know those requirements (as opposed to building a feature flag around these requirements and then updating those requirements with the feature flag). In addition, there was a bug with this implementation where it was not properly displaying ratings when there was no network connection, and you were viewing a subscribed podcast's details for the first time (it was interpreting the absence of a rating as "not enough ratings").

internal ref: p1697469972455619-slack-C028JAG44VD

> **Note**
> This is targeting the `release/7.50` branch. I don't think we need to rush out a new beta for this though.

## Testing Instructions
1. Install the app
2. View the details (including the ratings stars) on the podcast screen of a podcast with ratings 
3. View the details of a podcast without ratings (for example, The Big Dig).
4. Turn on airplane mode
5. Go to one of your subscribed podcasts that should have ratings but that you haven't viewed the details of yet and view the details of the podcast.
6. Observe that no rating is shown.

## Screenshot

![image](https://github.com/Automattic/pocket-casts-android/assets/4656348/dd13d2b0-b46c-40a0-bbd1-6bf31203d092)

## Checklist
